CMakeLists.txt

```diff
@@ -263,16 +269,17 @@ else()
 endif()
 
 if(MINGW)
-  target_link_libraries(SDL2 PRIVATE -nostdlib)
```

Comment:
Why are we removing this?

Reply:
I'm trying things out, since I think I saw an error about missing libgcc somewhre.
And SDL3 is not doing `-nostdlib` either.

Reply:
Missing libgcc is most possibly because clang has its own as compiler_rt (libclang_rt ??)

Reply:
`-nostdlib` turned out to be needed for avoiding duplicate `_DllMainCRTStartup` symbol errors.